services/ml/compilation_delegate_ie.cc

```diff
@@ -106,7 +108,15 @@ int32_t CompilationDelegateIe::Compile() {
 int32_t CompilationDelegateIe::CreateExecution(
     std::unique_ptr<mojom::Execution>& execution,
     mojom::ExecutionInitParamsPtr params) {
-  execution = std::make_unique<ExecutionImplIe>(this, std::move(params));
+  float input_scale = 1.0;
+  if (compilation_->GetPreference() == mojom::PREFER_ULTRA_LOW_POWER) {
+    const mojom::ModelInfoPtr& model = compilation_->GetModel();
+    DCHECK(model->inputs.size() == 1);
```

Comment:
For `ultra-low-power`, can the input size of input be only 1?

Reply:
Yes, GNA plugin only support one input seeing https://github.com/opencv/dldt/blob/2020/inference-engine/src/gna_plugin/gna_plugin.cpp#L1011.

Reply:
Good to know. Could you return a compilation error for multiple inputs?